docs/sources/configure-server/profile-cli.md

```diff
@@ -0,0 +1,234 @@
+---
+aliases:
+  - /docs/phlare/latest/operators-guide/configuring/profile-cli/
+  - /docs/phlare/latest/configure-server/profile-cli/
+description: Getting started with the profile CLI tool.
+menuTitle: Profile CLI
+title: Profile CLI
+weight: 80
+---
+
+# Profile CLI
+
+`profilecli` is a command-line utility that enables various productivity flows such as:
+- interacting with a running Pyroscope server to upload profiles manually, run queries, etc.
+- inspecting [Parquet](https://parquet.apache.org/docs/) files.
+
+> Hint: Use the `help` command (`profilecli help`) to get a full list of capabilities as well as additional help information.
+
+## Installation
+
+### MacOS
+
+```bash
+brew install pyroscope-io/brew/profilecli
+```
+
+### Other
+
+Download the `profilecli`` release asset from https://github.com/grafana/pyroscope/releases/latest for your operating system and architecture and make it executable.
+
+For example, for Linux with the AMD64 architecture:
+
+```bash
+curl -fL https://github.com/grafana/pyroscope/releases/download/v1.0.0/profilecli_1.0.0_linux_amd64.tar.gz | tar xvz
```

Comment:
Should we point to 1.1.5 ?

Reply:
did it for now, though this will get out of date quickly unless we generate these parts of the docs